arm-storageimportexport/2016-11-01/swagger/storageimportexport.json

```diff
@@ -0,0 +1,1194 @@
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LocationsListResult"
+            }
+          },
+          "default": {
+            "description": "An error occurs.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ImportExport/locations/{locationName}": {
+      "get": {
+        "tags": [
+          "StorageImportExport"
+        ],
+        "operationId": "GetLocation",
+        "description": "Gets a location to which you can ship the disks associated with an import or export job. A location is an Azure region.",
+        "parameters": [
+          {
+            "name": "locationName",
+            "in": "path",
+            "description": "The name of the location. For example, “West US” or “westus”.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/Location"
+            }
+          },
+          "default": {
+            "description": "An error occurs.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ImportExport/jobs": {
+      "get": {
+        "tags": [
+          "StorageImportExport"
+        ],
+        "operationId": "Jobs_ListBySubscription",
+        "description": "Gets all the active and completed import/export jobs in a subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "description": "An integer value that specifies how many jobs at most should be returned. The value cannot exceed 100.",
+            "required": false,
+            "type": "integer",
+            "maximum": 100,
+            "minimum": 1
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "Can be used to restrict the results to certain conditions. The following possible values can be used with $filter: 1) $filter=type eq '{type}'; 2) $filter=trackingnumber eq '{trackingnumber}'; 3) $filter=state eq '{state}'; 4) Logical and combination of the above, for example: $filter=type eq 'Import' and state eq 'Transferring'. Valid values for type are Import and Export. Valid values for state are Creating, Shipping, Received, Transferring, Packaging, Closed, and Completed.",
```

Comment:
From this comment it looks like the "state" property has a set of valid values. This makes state to be an enum candidate. Can you please add the enum array and the x-ms-enum extension to the state property. Btw, I found MutableJobProperties, JobProperties and DriveStatus all of these models have a state property.  So the state belongs to a Job or DriveStatus?